packages/web/package.json

```diff
@@ -29,20 +29,20 @@
     "electron:localhost": "electron . localhost",
     "pack": "electron-builder --dir",
     "dist": "node ./scripts/dist.js",
-    "dist:all": "turbo run dist -- --mac --win --linux",
-    "dist:mac": "turbo run dist -- --mac",
-    "dist:win": "turbo run dist -- --win",
-    "dist:linux": "turbo run dist -- --linux",
-    "dist-publish": "turbo run dist -- --mac --win --linux --publish always",
-    "dist-publish-production": "turbo run dist -- --mac --win --linux --publish always --env production",
-    "dist:mac-publish": "turbo run dist -- --mac --publish always",
-    "dist:win-publish": "turbo run dist -- --win --publish always",
-    "dist:linux-publish": "turbo run dist -- --linux --publish always",
-    "dist:win:linux-publish": "turbo run dist -- --win --linux --publish always",
-    "dist:mac-publish-production": "turbo run dist -- --mac --publish always --env production",
-    "dist:win-publish-production": "turbo run dist -- --win --publish always --env production",
-    "dist:linux-publish-production": "turbo run dist -- --linux --publish always --env production",
-    "dist:win:linux-publish-production": "turbo run dist -- --win --linux --publish always --env production",
+    "dist:all": "npm run dist -- --mac --win --linux",
+    "dist:mac": "npm run dist -- --mac",
+    "dist:win": "npm run dist -- --win",
+    "dist:linux": "npm run dist -- --linux",
+    "dist-publish": "npm run dist -- --mac --win --linux --publish always",
+    "dist-publish-production": "npm run dist -- --mac --win --linux --publish always --env production",
+    "dist:mac-publish": "npm run dist -- --mac --publish always",
+    "dist:win-publish": "npm run dist -- --win --publish always",
+    "dist:linux-publish": "npm run dist -- --linux --publish always",
+    "dist:win:linux-publish": "npm run dist -- --win --linux --publish always",
+    "dist:mac-publish-production": "npm run dist -- --mac --publish always --env production",
+    "dist:win-publish-production": "npm run dist -- --win --publish always --env production",
+    "dist:linux-publish-production": "npm run dist -- --linux --publish always --env production",
+    "dist:win:linux-publish-production": "npm run dist -- --win --linux --publish always --env production",
```

Comment:
This seems to work, kinda gross:

```suggestion
    "dist:all": "../../node_modules/.bin/turbo run dist -- --mac --win --linux",
    "dist:mac": "../../node_modules/.bin/turbo run dist -- --mac",
    "dist:win": "../../node_modules/.bin/turbo run dist -- --win",
    "dist:linux": "../../node_modules/.bin/turbo run dist -- --linux",
    "dist-publish": "../../node_modules/.bin/turbo run dist -- --mac --win --linux --publish always",
    "dist-publish-production": "../../node_modules/.bin/turbo run dist -- --mac --win --linux --publish always --env production",
    "dist:mac-publish": "../../node_modules/.bin/turbo run dist -- --mac --publish always",
    "dist:win-publish": "../../node_modules/.bin/turbo run dist -- --win --publish always",
    "dist:linux-publish": "../../node_modules/.bin/turbo run dist -- --linux --publish always",
    "dist:win:linux-publish": "../../node_modules/.bin/turbo run dist -- --win --linux --publish always",
    "dist:mac-publish-production": "../../node_modules/.bin/turbo run dist -- --mac --publish always --env production",
    "dist:win-publish-production": "../../node_modules/.bin/turbo run dist -- --win --publish always --env production",
    "dist:linux-publish-production": "../../node_modules/.bin/turbo run dist -- --linux --publish always --env production",
    "dist:win:linux-publish-production": "../../node_modules/.bin/turbo run dist -- --win --linux --publish always --env production",
```

We could also probably just reinstall turbo like you mentioned in the slack thread